content/seismic/raw_refraction_data.html

```diff
@@ -29,7 +29,7 @@
 pWin=window.open(urlstring,winName, "toolbar=0,scrollbars=1,directories=0,resizable=1,height=" + height + ",width=" + width + ",status=0"); 
 }
 
-MM_preloadImages('./../../_images/firstbreak2.gif','./../../_images/firstbreak3.gif','./../../_images/firstbreak1.gif');
+MM_preloadImages('./images/firstbreak2.gif','./images/firstbreak3.gif','./images/firstbreak1.gif');
```

Comment:
My understanding is that the figure using that radio button should be removed completely. That was my reasoning for leaving #112  open. This issue and that one can be closed when I update http://gpg.geosci.xyz/content/seismic/seismic_data.html